checkout_build_install_llvm.sh

```diff
@@ -28,11 +28,11 @@ WORK_DIR=$PWD
 mkdir -p $WORK_DIR/src
 
 # Checkout
-cd $WORK_DIR/src && git clone --depth 1 http://llvm.org/git/llvm.git
-cd $WORK_DIR/src/llvm/tools && git clone --depth 1 http://llvm.org/git/clang.git
-cd $WORK_DIR/src/llvm/projects && git clone --depth 1 http://llvm.org/git/compiler-rt.git
-cd $WORK_DIR/src/llvm/projects && git clone --depth 1 http://llvm.org/git/libcxx.git
-cd $WORK_DIR/src/llvm/projects && git clone --depth 1 http://llvm.org/git/libcxxabi.git
+cd $WORK_DIR/src && git clone --depth 1 https://github.com/llvm-mirror/llvm.git
```

Comment:
Thanks for the patch. https://github.com/llvm-mirror is an unofficial LLVM mirror. The official repo is https://github.com/llvm/llvm-project

Reply:
This is still wrong.